diff -urN kollins-0.2.orig/src/Makefile.am kollins-0.2/src/Makefile.am
--- kollins-0.2.orig/src/Makefile.am	Fri Aug  8 14:18:39 2003
+++ kollins-0.2/src/Makefile.am	Tue Sep 30 16:34:41 2003
@@ -25,7 +25,7 @@
 kollins_LDADD = $(LIB_KDEUI) $(LIB_KIO) $(LIB_KDEPRINT)
 
 # this is where the desktop file will go 
-shelldesktopdir   = $(kde_appsdir)/Utilities
+shelldesktopdir   = $(kde_appsdir)/Scientific
 shelldesktop_DATA = kollins.desktop
 
 # this is where the shell's XML-GUI resource file goes
diff -urN kollins-0.2.orig/src/Makefile.in kollins-0.2/src/Makefile.in
--- kollins-0.2.orig/src/Makefile.in	Sat Sep 13 13:42:56 2003
+++ kollins-0.2/src/Makefile.in	Tue Sep 30 16:35:34 2003
@@ -261,7 +261,7 @@
 kollins_LDADD = $(LIB_KDEUI) $(LIB_KIO) $(LIB_KDEPRINT)
 
 # this is where the desktop file will go 
-shelldesktopdir = $(kde_appsdir)/Utilities
+shelldesktopdir = $(kde_appsdir)/Scientific
 shelldesktop_DATA = kollins.desktop
 
 # this is where the shell's XML-GUI resource file goes
